NextcloudTalk/NCRoomsManager.m

```diff
@@ -807,10 +807,9 @@ - (void)joinOrCreateChatWithURL:(NSNotification *)notification
     NSString *server = [NCUtils valueForKey:@"server" fromQueryItems:queryItems];
     NSString *user = [NCUtils valueForKey:@"user" fromQueryItems:queryItems];
     NSString *withUser = [NCUtils valueForKey:@"withUser" fromQueryItems:queryItems];
-    NSString *accountId = [[NCDatabaseManager sharedInstance] accountIdForUser:user inServer:server];
-    TalkAccount *account = [[NCDatabaseManager sharedInstance] talkAccountForAccountId:accountId];
+    TalkAccount *account = [[NCDatabaseManager sharedInstance] talkAccountForUserId:user inServer:server];
     
-    [self checkForAccountChange:accountId];
+    [self checkForAccountChange:account.accountId];
```

Comment:
*Technically* `account` can still return `nil`, right? (tho improbable, since it *should've been* checked before) 
Would it be possible to send the account directly via the `userInfo` of the notification?

Reply:
Modified it. Now `accountId` and `withUser` are sent in `userInfo`.
Let me know what do you think @thisIsTheFoxe ;)

Reply:
Looking good, thanks.. I also like the alert reacting 👍